### PR TITLE
Make draft look like the next version

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description
supports: [https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1881](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1881)

I'll add a few notes to the code here in github.

requirements:

 * Call out: make it visually stand out (people are missing this)
 ** Add icon
 * Version info under the form title when created
 ** User wants to see version information right away (not draft)
 * Draft -> just next version (start from version 1, no draft)
 * Edit, Export, delete (draft), button on the version history table
 * Form Design title (on Manage Form page) & Team Management page
 
See screenshots in Jira ticket [https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1881](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1881)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

it would be good if the 'last updated' date in the form design history table was accurate. its to do with the way we're updating the db table with createDraft.  out of scope i guess.

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-142

File: app/frontend/src/components/forms/manage/ManageVersions.vue [hunk 1]
```diff
@@ -163,12 +263,28 @@ export default {
       return this.drafts && this.drafts.length > 0;
     },
     versionList() {
-      return this.form ? this.form.versions : [];
+      if(this.hasDraft){
```
Comment (TimCsaky) [inline app/frontend/src/components/forms/manage/ManageVersions.vue:266]: ..this was my idea to merge the draft object into the array of versions.
basically formatting like the version object but with 'draft: true' property.

File: app/frontend/src/components/forms/manage/ManageVersions.vue [hunk 2]
```diff
@@ -1,39 +1,38 @@
...
-      </template>
-
       <!-- Version  -->
       <template #[`item.version`]="{ item }">
         <router-link
-          :to="{
-            name: 'FormPreview',
-            query: { f: item.formId, v: item.id },
-          }"
+          :to=" item.isDraft ? { name: 'FormPreview', query: { f: item.formId, d: item.id } } : { name: 'FormPreview', query: { f: item.formId, v: item.id } }"
```
Comment (TimCsaky) [inline app/frontend/src/components/forms/manage/ManageVersions.vue:20]: preview url has parameters v for version d for draft

File: app/frontend/src/components/forms/manage/ManageVersions.vue [hunk 3]
```diff
@@ -103,54 +159,82 @@
...
+          sortable: false,
+        },
+        {
+          text: '',
+          value: 'export',
+          filterable: false,
+          sortable: false,
+        },
+        {
+          text: '',
```
Comment (TimCsaky) [inline app/frontend/src/components/forms/manage/ManageVersions.vue:219]: i could put all these actions in one column (?)

File: app/frontend/src/components/forms/manage/ManageVersions.vue [hunk 4]
```diff
@@ -103,54 +159,82 @@
...
+      formSchema: {
+        display: 'form',
+        type: 'form',
+        components: [],
+      },
       publishOpts: {
         publishing: true,
         version: '',
-        versionId: '',
+        id: '',
```
Comment (TimCsaky) [inline app/frontend/src/components/forms/manage/ManageVersions.vue:233]: i've relabeled this property so it makes sense as an identifier for both the 'version ID' and the 'draft ID'

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-142

**Review by jujaga — APPROVED** [review]

**Review by loneil — APPROVED** [review]
Looking great!

**Pull request merged**